hdeps/cli.py

```diff
@@ -155,7 +155,9 @@ def main(
 
     have_versions: Dict[CanonicalName, str] = {}
     for h in have:
-        k, _, v = h.partition("==")
+        k, op, v = h.partition("==")
+        if op != "==":
+            raise click.ClickException(f"Invalid format for --have: {h!r}")
```

Comment:
So the only specifier allowed in `--have` is `==` and nothing else? 

I thought in your demo yesterday you showed a case where you could do `--have "urllib3<2.01` or something similar?

Reply:
I might have, but that would only have worked as an argument, not with `--have`.

The idea was is that `--have` takes concrete versions and represents what's already in your venv, or what's checked in to your monorepo.